tests/core/test_playback.py

```diff
@@ -214,7 +211,33 @@ def test_get_current_track_next(self):
         self.replay_events()
 
         current_track = self.core.playback.get_current_track()
-        self.assertEqual(current_track, self.tracks[1])
+        assert current_track == self.tracks[1]
+
+    @pytest.mark.parametrize('repeat,random,single,consume,index,result', [
```

Comment:
I recently learned myself that spaces are allowed in this string. It adds a bit to its readability as the equivalent of an argument list.